src/subdomains/core/buy-crypto/process/services/buy-crypto-notification.service.ts

```diff
@@ -86,6 +88,8 @@ export class BuyCryptoNotificationService {
     for (const entity of entities) {
       try {
         if (entity.userData.mail) {
+          const mailWarningWalletIds = await this.settingService.getObj<number[]>('mailWarningWalletIds', []);
+
```

Comment:
Why not store it in the wallet table?

Reply:
As an additional column? Yeah could do that, I think 👍 
I thought it would not be so much wallets, but maybe there getting more which need this warning 😁

Reply:
Yes, I would prefer this